src/turing_model.jl

```diff
@@ -90,11 +100,57 @@ function turing_model(
     if standardize
         μ_X, σ_X, X = standardize_predictors(X)
         μ_y, σ_y, y = standardize_predictors(y)
+        if !isnothing(Z)
+            #TODO: implement random-effects slope
+            throw(
+                ArgumentError(
+                    "TuringGLM currently does not support random-effects for slope terms"
+                ),
+            )
+        end
     end
 
     # Random-Effects Conditionals
     if has_ranef(formula)
-        # TODO
+        if priors isa DefaultPrior
+            custom_prior = CustomPrior(
+                TDist(3), LocationScale(median(y), mad(y), TDist(3)), nothing
+            )
+        else
+            custom_prior = priors
+        end
+        intercept_ranef = intercept_per_ranef(ranef(formula))
+        group_var = first(ranef(formula)).rhs
```

Comment:
Just to check. Should this first be replaced by `only`?

Reply:
Currently yes. But for the future we would want to support multiple grouping variables.